pico_w/access_point/picow_access_point.c

```diff
@@ -6,86 +6,236 @@
 
 #include <string.h>
 
-#include "pico/stdlib.h"
 #include "pico/cyw43_arch.h"
+#include "pico/stdlib.h"
 
 #include "lwip/pbuf.h"
 #include "lwip/tcp.h"
 
 #include "dhcpserver.h"
-
-#ifndef USE_LED
-#define USE_LED 1
-#endif
+#include "dnsserver.h"
 
 #define TCP_PORT 80
 #define DEBUG_printf printf
+#define POLL_TIME_S 5
+#define HTTP_GET "GET"
+#define HTTP_RESPONSE_HEADERS "HTTP/1.1 %d OK\nContent-Length: %d\nContent-Type: text/html; charset=utf-8\nConnection: close\n\n"
+#define LED_TEST_BODY "<html><body><h1>Hello from Pico.</h1><p>Led is %s</p><p><a href=\"?led=%d\">Turn led %s</a></body></html>"
```

Comment:
Is it worth tweaking this to "Hello from Pico W". ? (for consistency with e.g. the MicroPython examples)